validate/validate.go

```diff
@@ -0,0 +1,574 @@
+package validate
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"reflect"
+	"strings"
+	"unicode"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/blang/semver"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+var (
+	defaultRlimits = []string{
+		"RLIMIT_CPU",
+		"RLIMIT_FSIZE",
+		"RLIMIT_DATA",
+		"RLIMIT_STACK",
+		"RLIMIT_CORE",
+		"RLIMIT_RSS",
+		"RLIMIT_NPROC",
+		"RLIMIT_NOFILE",
+		"RLIMIT_MEMLOCK",
+		"RLIMIT_AS",
+		"RLIMIT_LOCKS",
+		"RLIMIT_SIGPENDING",
+		"RLIMIT_MSGQUEUE",
+		"RLIMIT_NICE",
+		"RLIMIT_RTPRIO",
+		"RLIMIT_RTTIME",
+	}
+	defaultCaps = []string{
+		"CAP_CHOWN",
+		"CAP_DAC_OVERRIDE",
+		"CAP_FSETID",
+		"CAP_FOWNER",
+		"CAP_MKNOD",
+		"CAP_NET_RAW",
+		"CAP_SETGID",
+		"CAP_SETUID",
+		"CAP_SETFCAP",
+		"CAP_SETPCAP",
+		"CAP_NET_BIND_SERVICE",
+		"CAP_SYS_CHROOT",
+		"CAP_KILL",
+		"CAP_AUDIT_WRITE",
+	}
+)
+
+type Validator struct {
+	spec         rspec.Spec
+	rootfs       string
```

Comment:
Now I understand, you want to remove 'bundle' concept :)
But since 'BundlePath' is the entry to a user in current runtime-spec, you case is better in this way  (at least now).:
```
  for _, bundlePath := range bundles {
     v, _ := NewValidatorFromPath( bundlePath, true)
    msg = append(msgs, v.CheckAll())
  }
```

So I prefer my current commit#06163fe.